code/game/objects/items.dm

```diff
@@ -244,6 +244,10 @@
 	if(flags_item & DELONDROP)
 		qdel(src)
 
+///Called whenever an item is unequipped to a new loc (IE, not when the item ends up in the hands)
+/obj/item/proc/removed_from_inventory(mob/user)
```

Comment:
Add a signal for this proc to send
COMSIG_REMOVED_FROM_INVENTORY or something, would be really good if anyone wanted to know when a item is removed from inventory.